### PR TITLE
Xemko naming conventions and server/ readme file

### PR DESCRIPTION
**Pull request by Xemko**
Xemko naming conventions and server/ readme file

### CODE REVIEW

File: apps/server/readme.md
Comment (tgcaspa) [inline apps/server/readme.md:1]: Rename it to README.md

File: apps/server/readme.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# Plant-application
+
+## Installation
+
+1. Install the latest LTS version of Node.js from the official website: (https://nodejs.org)
```
Comment (tgcaspa) [inline apps/server/readme.md]: Q: what is the minimum of your NOde.js version? please mention. I'm using 18.19.0

Comment (tgcaspa) [reply apps/server/readme.md]: I can upgrade if reqired

Comment (tgcaspa) [reply apps/server/readme.md]: I can upgrade if required

Comment (Xemko) [reply apps/server/readme.md]: mine v18.18.0

Comment (Xemko) [reply apps/server/readme.md]: approve give chel

**Review by tgcaspa — APPROVED** [review]

**Pull request merged**